components/StyledLink/index.js

```diff
@@ -1,15 +1,18 @@
 import styled from "styled-components";
 import Link from "next/link";
 export const StyledLink = styled(Link)`
-  display: inline-block;
-  padding: 5px 10px;
+  display: flex;
+  justify-content: center;
+  padding: 5px 5px;
   border-radius: 5px;
   background-color: #b4ccb9;
   color: #f76d57;
-  text-decoration: none;
-  font-weight: bold;
   text-align: center;
-  border: none;
+  border: solid #45aab8;
+  height: 30px;
+  width: 100px;
+  line-height: 1.4;
+  margin: auto;
```

Comment:
The stylings for your `StyledLink` and your `HeaderLink` look almost the same. You can think about working with variants. 

An example for a button would look like this:

```jsx
import styled, { css } from "styled-components";

export const StyledButton = styled.button`
  background-color: ${({ disabled }) => (disabled ? "black" : "lightsalmon")};
  padding: 0.8rem;
  border-radius: 0.6rem;
  color: black;
  text-decoration: none;
  font-weight: bold;
  border: none;
  font-size: inherit;

  ${({ variant }) =>
    variant === "delete" &&
    css`
      background-color: firebrick;
      color: white;
    `}
`;
```

and when you use the button in another component you can set the prop variant:

```jsx
...
        <StyledButton
          onClick={onDelete}
          type="button"
          variant="delete"
        >
          Delete
        </StyledButton>
...
```